src/classes/RLLP_OppRollup.cls

```diff
@@ -363,7 +363,15 @@ public without sharing class RLLP_OppRollup {
             }
         }         
 
-        update accountsToUpdate.values();     
+        /**Randi - Changes for Accounts - can I pass in account type?
```

Comment:
Is this comment a to-do, a note, or just something that can be removed?

Reply:
I did want to discuss if you had any suggestions on how to pass in the record type so we could include that in the error messaging, but otherwise I'll just remove this.

Reply:
Is Record Type the SObjectType? If so, you can get that from the record Id using .getSObjectType.

Reply:
Right, I guess I'm just wondering if this will always be a list of the same object type? If so, I could just query the first object and call it good, otherwise I'll need to check the whole list to make sure, and maybe break if a different object type is found.

Reply:
Good question. For the most part, all of our dml operations are on the same object, so I don't see any issue in just using the first Id with `getSObjectType().getDescribe().getName()`.